src/components/OperationTag.js

```diff
@@ -78,18 +80,19 @@ export default class OperationTag extends React.Component {
               }
           </p>
 
-          <button
-            className="expand-operation"
-            title={showTag ? "Collapse operation": "Expand operation"}
-            onClick={() => layoutActions.show(isShownKey, !showTag)}
-            aria-expanded={showTag}
-            tabIndex={-1}
-          >
-            <svg className="arrow" width="20" height="20">
-              <use href={showTag ? "#large-arrow-down" : "#large-arrow"} xlinkHref={showTag ? "#large-arrow-down" : "#large-arrow"} />
-            </svg>
-          </button>
-        </h1>
+            <button
+              className="expand-operation"
+              title={showTag ? "Collapse operation": "Expand operation"}
+              onClick={() => layoutActions.show(isShownKey, !showTag)}
+              aria-expanded={showTag}
+              tabIndex={-1}
+              >
+
+              <svg className="arrow" width="20" height="20">
+                <use href={showTag ? "#large-arrow-down" : "#large-arrow"} xlinkHref={showTag ? "#large-arrow-down" : "#large-arrow"} />
+              </svg>
+            </button>
```

Comment:
Is anything from 83-94 changing? or is it just formatting? Hard to tell from the diff

Reply:
You can view the PR ignoring whitespace [like this](https://github.com/Kong/swagger-ui-kong-theme/pull/119/files?diff=split&w=1), but it's just whitespace, thankfully!